### PR TITLE
Guides variant: K5

### PR DESCRIPTION
**Pull request by breville**
With this change, AI for Oceans supports a new variant of guides for some new K-5 curriculum.  This means that the tutorial can still run unchanged from its original form as an Hour of Code tutorial, but the curriculum team can author a new set of guide messages, for the same modes as before, for new K-5 curriculum.

For now, this can be authored by running locally with URLs like the following:
- http://localhost:8080 runs the original Hour of Code guides.
- http://localhost:8080/?guides=K5 runs with the new K5 guides.

The original Hour of Code guides are now in `guidesHoc.js`, and should remain unchanged for now.  The new guides duplicate this set, with one example text change, in `guidesK5.js`, which can now be modified by the curriculum team.

### CODE REVIEW

File: src/oceans/models/guide.js [hunk 1]
```diff
@@ -479,7 +10,9 @@ const getCurrentGuide = () => {
 
   const state = getState();
 
-  for (const guide of guides) {
+  const currentGuides = queryStrFor('guides') === 'K5' ? guidesK5 : guidesHoc;
```
Comment (bethanyaconnor) [inline src/oceans/models/guide.js:13]: I assume this will be configurable on the studio side?

**Review by bethanyaconnor — COMMENTED** [review]
What is the imagined way for curriculum writers to update this. Strings really need to be added to https://github.com/code-dot-org/ml-activities/blob/main/i18n/oceans.json, then use the `I18n.t` function to call them. Then we'll need to export this package. I assume this will be a one-time update as a result?

File: src/oceans/models/guide.js (hunk 1 above)
Comment (breville) [reply src/oceans/models/guide.js]: Great question.  I just updated this PR to make this a parameter for `initAll`, and then we will be able to configure this on the studio side.

**Comment by ebeastlake** [comment]
Acknowledged/seems reasonable, pending approval by @bethanyaconnor. Thanks, @breville!

**Review by bencodeorg — APPROVED** [review]
LGTM -- just so we don't lose it, what's the "small change" you made to the K5 guides?

**Comment by breville** [comment]
> LGTM -- just so we don't lose it, what's the "small change" you made to the K5 guides?

Oh, it's the test string [here](https://github.com/code-dot-org/ml-activities/pull/402/files#diff-c116fd73dcc5534b131136b95ae2506cc60b370f8bca5103e679d90a4742a140R154).

File: src/oceans/models/guidesK5.js [hunk 2]
```diff
@@ -0,0 +1,485 @@
...
+  },
+  {
+    id: "fishvtrash-pond-init3",
+    textFn: () => I18n.t("fishvtrash-pond-init3"),
+    when: { appMode: AppMode.FishVTrash, currentMode: Modes.Pond },
+    arrow: "LowerRight",
+  },
+  {
+    id: "creaturesvtrashdemo-predicting-init1",
+    textFn: () => 'This is a test string.',
```
Comment (breville) [inline src/oceans/models/guidesK5.js:154]: Here is an example unlocalized string that can be used to confirm which variant is showing.

**Pull request merged**